### PR TITLE
Add values to Readme.md

### PR DESCRIPTION
**Pull request by DianaOlympos**
This is a first try after some time searching the literature about what these value were.

I will post comments inline about the things i am not sure about.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,13 @@
...
-  ]
-end
-```
-
-Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
-and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
-be found at [https://hexdocs.pm/gravitas](https://hexdocs.pm/gravitas).
+The values that guide the design of Gravitas are:
 
+- *Observability*: You should be able to understand what Gravitas is doing easily. Users should be able to maintain an understanding of the problem from Gravitas' perspective. The user should be able to have a representation of the problem state shared with Gravitas and a representation of both the activities of Gravitas and other users using it. Users need to be able to see what Gravitas is doing and what it will do next relative to its state. And this without need for overly taxing cognitive activity
```
Comment (DianaOlympos) [inline README.md]: This is really verbose, i think we should cut down, but i am not sure of what to keep and what to throw away yet.

File: README.md [hunk 2]
```diff
@@ -1,21 +1,13 @@
...
+- *Directability*: You need to be able to control and steer the direction Gravitas is going into when you recognize a need to intervene. And this without having to take full manual control. Gravitas should be flexible and able to take directions in the areas that needs supervision. And doing so while keeping the rest of its functionality active in an adapted way. As above, that need to be possible without overly taxing cognitive activity from the user.
+- *Learnability*: Gravitas should provide ways for the user to grow its expertise with it. Gravitas handle a complex problem state and is itself a complex system. We consider that moving our user from the first discovery of the system to being an expert in it is essential. That may be through experimentations in a simulated environment, documentations, talking with other experts, etc. This should be an integral part of the user experience of Gravitas.
```
Comment (DianaOlympos) [inline README.md]: The value is a really awful word by itself. Better idea ? Same thing, some of the explanation are understandable and efficient, but really not impressively well worded imho.

File: README.md [hunk 3]
```diff
@@ -1,21 +1,13 @@
...
+- *Safety*: Gravitas need to fill the highest profesional standard for safety, security and ethics. We recognise our place in the ecosystem of software. As much as possible, we provide fail-safe situations and sane defaults. We aim to make the safe path easy. If it has to choose between ressources and safety, Gravitas should balance its choice toward safety.
+- *Stability*: We acknowledge that numerous service and workflow will be built on top and around Gravitas. Gravitas will try to aim to not disrupt target without reasons. Gravitas will also try to provide a stable interface. Or at the minimum an easy path for upgrade.
```
Comment (DianaOlympos) [inline README.md]: I tried to translate what you said about it, but you may be able to deal with it better.

File: README.md (hunk 1 above)
Comment (MordecaiMalignatus) [reply README.md]: Maybe try something like `The state and actions of Gravitas should be visible, cause and consequent action taken should be visible and *understandable* to a user.`

File: README.md (hunk 2 above)
Comment (MordecaiMalignatus) [reply README.md]: "Ability to grow with the user", maybe?

File: README.md (hunk 3 above)
Comment (MordecaiMalignatus) [reply README.md]: I would go with something like 
>As a infrastructure, Gravitas' behaviour must be stable, and predictable. This means interfaces must be as stable as possible, upgrade paths must be easy, and Gravitas itself generally must bring a lot more benefit than it has cost to operate.

File: README.md [hunk 4]
```diff
@@ -1,21 +1,13 @@
...
+- *Learnability*: Gravitas should provide ways for the user to grow its expertise with it. Gravitas handle a complex problem state and is itself a complex system. We consider that moving our user from the first discovery of the system to being an expert in it is essential. That may be through experimentations in a simulated environment, documentations, talking with other experts, etc. This should be an integral part of the user experience of Gravitas.
+- *Safety*: Gravitas need to fill the highest profesional standard for safety, security and ethics. We recognise our place in the ecosystem of software. As much as possible, we provide fail-safe situations and sane defaults. We aim to make the safe path easy. If it has to choose between ressources and safety, Gravitas should balance its choice toward safety.
```
Comment (MordecaiMalignatus) [inline README.md]: Drop the "professional" from "highest professional standards" imho. It has been said too much in snake-oil context, so in my view it actually devalues the standards we strive towards.

**Review by MordecaiMalignatus — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -1,21 +1,15 @@
 # Gravitas
 
-**TODO: Add description**
+An agent to control your infrastructure.
 
-## Installation
+The main goal of Gravitas is to provide help in managing your cloud infrastructure, without overly taxing cognitive activity from the user.
```
Comment (MordecaiMalignatus) [inline README.md]:
```suggestion
The main goal of Gravitas is to provide help in managing your cloud infrastructure, without significant mental overhead for the user.
```

File: README.md [hunk 6]
```diff
@@ -1,21 +1,15 @@
...
-end
-```
-
-Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
-and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
-be found at [https://hexdocs.pm/gravitas](https://hexdocs.pm/gravitas).
+The values that guide the design of Gravitas are:
 
+- *Observability*: The state and actions of Gravitas should be visible and easy to understand. Reasons for a decision and consequent action taken should be visible and *understandable* to a user.
+- *Directability*: You need to be able to control and steer the direction Gravitas is going into, when you recognize a need to intervene. Gravitas should be flexible and able to take directions in the areas that needs supervision, without the need for the user to take full control.
```
Comment (MordecaiMalignatus) [inline README.md]:
```suggestion
- *Directability*: You need to be able to control and steer the direction Gravitas is going into, when you recognize a need to intervene. Gravitas should be flexible and able to take directions in the areas that need supervision, without the need for the user to take full control.  
```

File: README.md [hunk 7]
```diff
@@ -1,21 +1,15 @@
...
 
+- *Observability*: The state and actions of Gravitas should be visible and easy to understand. Reasons for a decision and consequent action taken should be visible and *understandable* to a user.
+- *Directability*: You need to be able to control and steer the direction Gravitas is going into, when you recognize a need to intervene. Gravitas should be flexible and able to take directions in the areas that needs supervision, without the need for the user to take full control.  
+- *Learnability*: Gravitas should be easy to pick up. We should also allow the user to grow their expertise alongside use of Gravitas. Gravitas handle a complex problem state and is itself a complex system. We consider that we should accompany our user from the first discovery of the system to being an expert of it.
+- *Safety*: Gravitas need to fill the highest standard for safety, security and ethics. We recognise our place in the ecosystem of software. As much as possible, we provide fail-safe situations and sane defaults. We aim to make the safe path easy. If a choice have to be made between ressources and safety, Gravitas should balance its choice toward safety.
```
Comment (MordecaiMalignatus) [inline README.md]:
```suggestion
- *Safety*: Gravitas needs to fulfill the highest standard for safety, security and ethics. We recognise our place in the ecosystem of software. As much as possible, we provide fail-safe situations and sane defaults. We aim to make the safe path easy. If a choice have to be made between resources and safety, Gravitas should balance its choice towards safety.
```

File: README.md [hunk 8]
```diff
@@ -1,21 +1,15 @@
...
-```
-
-Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
-and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
-be found at [https://hexdocs.pm/gravitas](https://hexdocs.pm/gravitas).
+The values that guide the design of Gravitas are:
 
+- *Observability*: The state and actions of Gravitas should be visible and easy to understand. Reasons for a decision and consequent action taken should be visible and *understandable* to a user.
+- *Directability*: You need to be able to control and steer the direction Gravitas is going into, when you recognize a need to intervene. Gravitas should be flexible and able to take directions in the areas that needs supervision, without the need for the user to take full control.  
+- *Learnability*: Gravitas should be easy to pick up. We should also allow the user to grow their expertise alongside use of Gravitas. Gravitas handle a complex problem state and is itself a complex system. We consider that we should accompany our user from the first discovery of the system to being an expert of it.
```
Comment (MordecaiMalignatus) [inline README.md]:
```suggestion
- *Learnability*: Gravitas should be easy to pick up. We should also allow the user to grow their expertise alongside use of Gravitas. Gravitas handles a complex problem state and is itself a complex system. We consider that we should accompany our user from the first discovery of the system to being an expert of it.
```

**Review by MordecaiMalignatus — APPROVED** [review]

**Pull request merged**